<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistressCallClass</class>
 <widget class="QMainWindow" name="DistressCallClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DistressCall</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QTabWidget" name="ChardTab">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="modeATab">
       <attribute name="title">
        <string>Mode A</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="fileLocationLabel">
                 <property name="text">
                  <string>fileLocation</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="fileLocationLineEdit">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="myListLocationLabel">
                 <property name="text">
                  <string>myListLocation</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="myListLocationLineEdit">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QPushButton" name="fileLocationButton">
                 <property name="text">
                  <string>Add fileLocation</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="myListLocationButton">
                 <property name="text">
                  <string>Add myListLocation</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="EvidenceFileWidget"/>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="IdLabel">
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Id</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="IdLineEdit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="MeasurementLabel">
               <property name="text">
                <string>Measurement</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="MeasurementLineEdit"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="ImageClarityLevelLabel">
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text">
                <string>ImageClarityLevel</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="ImageClarityLevelLineEdit"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="QuantityLabel">
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Quantity</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="QuantityLineEdit"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="PhotographLabel">
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Photograph</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="PhotographLineEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="AddButton">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="DeleteButton">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="UpdateButton">
               <property name="text">
                <string>Update</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="UndoButton">
            <property name="text">
             <string>Undo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RedoButton">
            <property name="text">
             <string>Redo</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="modeBTab">
       <attribute name="title">
        <string>Mode B</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QListWidget" name="currentEvidenceFileWidget">
              <property name="maximumSize">
               <size>
                <width>674</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QPushButton" name="SaveIdButton">
                <property name="text">
                 <string>Save to myList</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="NextButton">
                <property name="text">
                 <string>Next</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="myListWidget"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="MeasurementFilterLabel">
                <property name="text">
                 <string>Measurement</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="MeasurementFilterLineEdit"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="QuantityFilteredLabel">
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Minimum Quantity</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="QuantityFilterLineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="FilteredListButton">
              <property name="text">
               <string>Filtered list</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="MyListButton">
              <property name="text">
               <string>MyList As File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="MyListTableButton">
              <property name="text">
               <string>MyList As Table</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="UndoMyListButton">
            <property name="text">
             <string>Undo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RedoMyListButton">
            <property name="text">
             <string>Redo</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DistressCall.qrc"/>
 </resources>
 <connections/>
</ui>
